src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -96,6 +94,13 @@ public sealed class ProjectInfo
         /// </summary>
         public Type HostObjectType { get; }
 
+        /// <summary>
+        /// True if project information is complete. In some workspace hosts, it is possible
+        /// a project only has partial information. In such cases, a project might not have all
+        /// information on its files or references.
```

Comment:
When does a project not have all information on its files?

Also, as above, i think we need a better name.  I think HasSuccessfullyLoadedAllReferences would make it very clear what this means.  As this is internal, i don't mind having a nice full name to make the code clearer.